File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    lux = luy = 51
    rdx = rdy = -1
    
    n, m = len(wallpaper), len(wallpaper[0])
    
    for i in range(n):
        for j in range(m):
            if wallpaper[i][j] != '.':
                lux = min(lux, i)
                luy = min(luy, j)
                rdx = i + 1
                rdy = max(rdy, j + 1)
            
    return lux, luy, rdx, rdy

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    
    public int[] solution(String[] wallpaper) {
        int lux = 51;
        int luy = 51;
        int rdx = -1;
        int rdy = -1;

        int n = wallpaper.length;
        int m = wallpaper[0].length();

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {
                if (wallpaper[i].charAt(j) == '.') {
                    continue;
                }

                lux = Math.min(lux, i);
                luy = Math.min(luy, j);
                rdx = i + 1;
                rdy = Math.max(rdy, j + 1);
            }
        }

        return new int[]{lux, luy, rdx, rdy};
    }
}